lib/preprocessors/glimmer.js

```diff
@@ -14,6 +14,22 @@ function escapeRegExp(string) {
   return string.replace(/[$()*+.?[\\\]^{|}]/g, '\\$&'); // $& means the whole matched string
 }
 
+function arrayEq(arr1, arr2) {
+  let returnVal = true;
+  for (const [idx, val1] of arr1.entries()) {
+    if (val1 !== arr2[idx]) {
+      returnVal = false;
```

Comment:
if you early return here the array won't need to keep iterating.
Just a lil optimization <3